src/app/alert/angular/alert-angular-not-closable.demo.html

```diff
@@ -21,16 +21,16 @@
                         <div class="alert-actions">
                             <a class="alert-action" href="javascript://">Acknowledge</a>
                             <a class="alert-action" href="javascript://">Reset</a>
-                            <clr-dropdown [clrMenuPosition]="'bottom-right'">
+                            <clr-dropdown>
                                 <button class="dropdown-toggle" clrDropdownToggle>
                                     More Actions
                                     <clr-icon shape="caret down"></clr-icon>
                                 </button>
-                                <div class="dropdown-menu">
+                                <clr-dropdown-menu clrPosition="bottom-right">
                                     <a href="javascript://" class="dropdown-item" clrDropdownItem>Shutdown</a>
```

Comment:
This note is for me to go in and change all links in menu to buttons instead before 0.10.0